Description: Partially revert https://chromium.googlesource.com/chromium/third_party/ffmpeg.git/+/40d58062f36b89daa5a19826cb9d89a0b1fc6d35
 When building on zesty and older (clang 4.0), llvm's integrated assembler fails to build ffmpeg on armhf.
 Other architectures are not affected, and neither are builds done on artful or newer (clang 5.0).
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/third_party/ffmpeg/BUILD.gn
+++ b/third_party/ffmpeg/BUILD.gn
@@ -267,6 +267,9 @@ target(link_target_type, "ffmpeg_interna
       } else {
         asmflags += [ "-DHAVE_VFP_ARGS=0" ]
       }
+      if (is_clang) {
+        asmflags += [ "-fno-integrated-as" ]
+      }
     }
   }
 
